apps/web/components/Charts/ChartsCard/ChartsCard.treat.ts

```diff
@@ -1,16 +1,61 @@
-import { style, styleMap } from 'treat'
-import { theme } from '@island.is/island-ui/theme'
+import { style } from 'treat'
 
-export const card = style({
+export const frameWrapper = style({
   display: 'flex',
   height: '646px',
   width: '100%',
   flexDirection: 'column',
   boxSizing: 'border-box',
   minHeight: 124,
   textDecoration: 'none',
+  overflowX: 'scroll',
   position: 'relative',
+  // overflow: 'visible',
+  background: 'transparent',
+  outline: 'none',
   ':hover': {
     textDecoration: 'none',
   },
 })
+
+export const card = style({
+  display: 'flex',
```

Comment:
if we are using the Box component we should try to use the internal styles from the Box component for most of our styles

ex.
```jsx
<Box
  display="flex"
  flexDirection="column"
  flexGrow={1}
  alignItems="stretch"
  justifyContent="flexStart"
></Box>
```
this minimizes generated css as it will reuse the Box css

Reply:
this applies to all styles that will be using the box component